tox.ini

```diff
@@ -123,41 +152,62 @@ deps =
 	sphinxcontrib-spelling
 	pyenchant
 commands =
-	- sphinx-build {posargs:-E} -Q -b html docs dist/docs
-	sphinx-build -b spelling -w /dev/null docs dist/docs
+	- sphinx-build {posargs:-E} -Q -b html docs build/docs
+	sphinx-build -b spelling -w /dev/null docs build/docs
 
-[testenv:link]
+[testenv:check-docs-link]
 description = Check that the links written in documentation are valid.
 skip_install = true
 deps = -r{toxinidir}/docs/requirements.txt
 commands =
-	- sphinx-build {posargs:-E} -Q -b html docs dist/docs
-	sphinx-build -b linkcheck -w /dev/null docs dist/docs
+	- sphinx-build {posargs:-E} -Q -b html docs build/docs
+	sphinx-build -b linkcheck -w /dev/null docs build/docs
+
+[testenv:run-black]
+description = Run black tool on the code.
+basepython = python3.6
+skip_install = true
+deps =
+	black
+commands =
+	black --config config/black.toml {toxinidir}/src/appsettings
+
+[testenv:run-isort]
+description = Check the imports order.
+deps =
+	isort
+commands =
+	isort --settings-path config --apply --recursive src/appsettings tests setup.py
+
+[testenv:run-bumpversion]
+description = Check the imports order.
+deps =
+	bumpversion
+commands =
+	bumpversion --config-file config/bumpversion.conf {posargs}
```

Comment:
`config/bumpversion.conf` does not exist.

Reply:
Oops :fearful: